docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,113 @@
+# Syncing LDAP Groups to OpenShift Groups
+
+OpenShift cluster admins need a way to sync their OpenShift `Groups` to external records in an LDAP server. This requires that we have a way of retrieving the LDAP first-class user entries that are the members of a group, determining the OpenShift `Group` name for this collection, and updating the OpenShift record, removing it as necessary when the corresponding LDAP record is removed. Necessary information about the LDAP server set-up and schema will be provided by the user in a `yaml` configuration file and pointed to in the `oadm sync-groups` command.
```

Comment:
Don't use "it".  

We don't care about the ldap user object's "first class" status".